testing/jormungandr-testing-utils/resources/explorer/graphql/schema.graphql

```diff
@@ -1,6 +1,146 @@
 """Directs the executor to query only when the field exists."""
 directive @ifdef on FIELD
 
+type Block0Date {
+  block0_date: Int!
+}
+
+enum DiscriminationEnum {
+  PRODUCTION,
+  TEST,
+}
+
+type Discrimination {
+  discrimination: DiscriminationEnum!
+}
+
+enum ConsensusTypeEnum {
+  BFT,
+  GENESISPRAOS,
+}
+
+type ConsensusType {
+  consensus_type: ConsensusTypeEnum!
```

Comment:
I'm a bit curious on what's the reason for this level of indirection. Why not have the enum be called `ConsensusType`?

Reply:
`assync_graphql` has a problem, it is impossible to make a GraphQl `union` in the following way:
```

#[derive(Enum)]
enum Bar {
   SomeBar,
}


#[derive(Union)]
enum A {
    Foo(Bar)
}

```
Because for the GraphQL union trait it is needed that `Bar` will implement an GraphQL `Object` trait and the only way to workaround it is to wrapper `enum Bar` over other `struct Bar` like I have done here

Reply:
Hmmm, apparently the graphql spec doesn't support having nested unions. Interesting.

Maybe we could considering flattening things, but I think this is fine.